hazelcast/pom.xml

```diff
@@ -486,5 +486,16 @@
             <version>2.26.3</version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.openjdk.jmh/jmh-core -->
```

Comment:
I think you don't have to add it here, it is already added to the parent project (but please double-check).